anomalib/models/components/base/anomaly_module.py

```diff
@@ -155,15 +159,28 @@ def _collect_outputs(image_metric, pixel_metric, outputs):
     def _post_process(outputs):
         """Compute labels based on model predictions."""
         if "pred_scores" not in outputs and "anomaly_maps" in outputs:
+            # infer image scores from anomaly maps
             outputs["pred_scores"] = (
                 outputs["anomaly_maps"].reshape(outputs["anomaly_maps"].shape[0], -1).max(dim=1).values
             )
-
-    @staticmethod
-    def _outputs_to_cpu(output):
-        for key, value in output.items():
-            if isinstance(value, Tensor):
-                output[key] = value.cpu()
+        elif "pred_scores" not in outputs and "boxes_scores" in outputs:
+            # infer image score from bbox confidence scores
+            outputs["pred_scores"] = torch.stack([scores.max() for scores in outputs["boxes_scores"]])
+        if "pred_boxes" in outputs and "anomaly_maps" not in outputs:
```

Comment:
Can we have a new line above as it initially looks confusing to see an `if ` statement right after `elif`